fastlane/lib/fastlane/actions/actions_helper.rb

```diff
@@ -87,7 +87,7 @@ def self.load_external_actions(path)
           class_ref = Fastlane::Actions.const_get(class_name)
 
           if class_ref.respond_to?(:run)
-            UI.success "Successfully loaded custom action '#{file}'."
+            UI.success "Successfully loaded custom action '#{file}'." if $verbose
```

Comment:
Since it says that the external action was successfully loaded and is a valid action